### ISSUE TITLE
StringIO#puts does not print a newline character when not given any arguments

### ISSUE THREAD

**Issue by ntl** [ksss/mruby-stringio#6]
It looks like `StringIO#puts` prints a newline after every string argument given, but it does not handle the case where `#puts` is called without any arguments. MRI prints a newline in this case.

```ruby
string_io = StringIO.new
string_io.puts
# Will not pass currently
assert(string_io.string == "\n")
```

For the time being, I can work around it by passing an empty string, e.g. `#puts("")`

**Comment by ksss** [comment]
@ntl Thank you for reporting.
I fixed in ee5df6e4c70e85aa29ac4d50afbc76e0f6fdf236

**Comment by ntl** [comment]
Thanks @ksss !
